Clarify that LDAP will create Grafana users by default when they log in for the first time (#63391)
grafana/grafana: docs/sources/setup-grafana/configure-security/configure-authentication/ldap/index.md

In order to use LDAP integration you'll first need to enable LDAP in the [main config file]({{< relref "../../../configure-grafana/" >}}) as well as specify the path to the LDAP
specific configuration file (default: `/etc/grafana/ldap.toml`).

> Once LDAP has been enabled, the default behavior is for Grafana users to be created automatically upon successful LDAP authentication. If you would prefer for only existing Grafana users to be able to sign in, you may change `allow_sign_up` to `true` in the `[auth.ldap]` section.

Reviewer: ```suggestion
After enabling LDAP, the default behavior is for Grafana users to be created automatically upon successful LDAP authentication. If you prefer for only existing Grafana users to be able to sign in, you can change `allow_sign_up` to `true` in the `[auth.ldap]` section.
```
